data/json/items/ammo/84x246mm.json

```diff
@@ -31,7 +31,7 @@
     "//": "84mm HEDP 502 RS.",
     "price": "700 USD",
     "price_postapoc": "120 USD",
-    "name": { "str_sp": "84x245mm HEDP" },
+    "name": { "str": "84mm anti-materiel cartridge" },
```

Comment:
Nitpick: "Anti-material" is a pretty specific term that just isn't appropriate for a recoilless rifle round.